### PR TITLE
Remix tutorial (trivial, but I think we should have something)

### PR DESCRIPTION
**Pull request by qbzzt**
Closing: INT-275

### CODE REVIEW

File: remix/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+> :warning: We are currently in the process of upgrading to OVM 2.0, but that
+> is still work in progress (expected to end 11 NOV 2021). This tutorial is
+> already upgraded for OVM 2.0, but parts of it will change during the upgrade
+> process.
+
+   <!-- TEMP-OVM2.0 -->
+
+## Setup
+
+In OVM 2.0 there is no more need for the Remix Optimism plugin, please ignore it. All
```
Comment (K-Ho) [inline remix/README.md:19]: I would prefer not even referencing the plugin by name. 
```suggestion
In OVM 2.0, there is no need for any custom Remix plugins. All
```

File: remix/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+you need to do is specify [the network information](https://community.optimism.io/docs/infra/networks.html#optimistic-kovan).
+
+To use Remix:
+
+1. Log on with your wallet to Optimistic Ethereum (or Optimistic Kovan).
```
Comment (K-Ho) [inline remix/README.md:24]: Link to guide on how to do this or link out to chainid.link

**Review by K-Ho — DISMISSED** [review]

File: remix/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+   Optimistic Ethereum). You can do this using chainid.link:
+   - [Optimistic Kovan](https://chainid.link?network=optimism-kovan)
+   - [Optimistic Ethereum](https://chainid.link?network=optimism)
+
+1. Browse to [Remix](https://remix.ethereum.org/).
+1. Click the run icon (<img src="https://remix-ide.readthedocs.io/en/latest/_images/a-run-icon.png" height="24" valign="top" />).
+1. Select the Environment **Injected Web3 Provider**.
+1. Accept the connection in the wallet.
+
+That's it.
```
Comment (Nicca42) [inline remix/README.md:34]: Now you can start writing OVM contracts! "That's it!" feels a bit weird here.

**Review by Nicca42 — COMMENTED** [review]
Let me know what you think of the comments @qbzzt

**Comment by qbzzt** [comment]
> Let me know what you think of the comments @qbzzt

I think they're right, and I addressed them. Thank you.

**Review by Nicca42 — APPROVED** [review]
👏

**Pull request merged**